src/TelegramLog.php

```diff
@@ -84,6 +85,9 @@ public static function initialize(\Monolog\Logger $external_monolog = null)
      */
     public static function initErrorLog($path)
     {
+        if (empty($path)) {
```

Comment:
When comparing strings, it's better to compare `=== ''` or `!== ''`, because maybe I want my filename to be `0` (which is regarded as empty)
I noticed that this bug is in Monolog too... When I call the file `1` it works, `0` doesn't, because it probably also has an `empty()` check somewhere.